templates_test/main_test/crdb_main.tpl

```diff
@@ -0,0 +1,130 @@
+type pgTester struct {
```

Comment:
I notice this file is formatted very strangely, not like Go code (no tabs for example). Might be worth it to run gofmt on it once.

Reply:
Umm..., strange. Will take a look.